servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -74,10 +105,29 @@ public LoadBalancer<FilterableStreamingHttpLoadBalancedConnection> newLoadBalanc
         return rawFactory.newLoadBalancer(eventPublisher, connectionFactory, targetResource);
     }
 
-    @Override   // FIXME: 0.43 - remove deprecated method
+    @Override
     public FilterableStreamingHttpLoadBalancedConnection toLoadBalancedConnection(
-            final FilterableStreamingHttpConnection connection) {
-        return new DefaultFilterableStreamingHttpLoadBalancedConnection(connection);
+            final FilterableStreamingHttpConnection connection,
+            final ReservableRequestConcurrencyController concurrencyController,
+            @Nullable final ContextMap context) {
+
+        RequestTracker hostHealthIndicator = null;
+        if (context == null) {
+            LOGGER.debug("Context is null. In order for " + DefaultHttpLoadBalancerFactory.class.getSimpleName() +
```

Comment:
1. Consider adding assertion in case this can be surfaced by a test.
2. Consider increasing the log level to info. If it's unexpected, we should know about it.

Reply:
Until the new LB becomes the default, these can cause unnecessary noise if added in a higher log level. For current defaults, they aren't needed. We can do so when that changes.